Neos.EventSourcedNeosAdjustments/Classes/EventSourcedFrontController/EventSourcedNodeController.php

```diff
@@ -131,36 +125,33 @@ class EventSourcedNodeController extends ActionController
      * @throws \Neos\Neos\Exception
      * @Flow\SkipCsrfProtection We need to skip CSRF protection here because this action could be called with unsafe requests from widgets or plugins that are rendered on the node - For those the CSRF token is validated on the sub-request, so it is safe to be skipped here
      */
-    public function showAction(NodeAddress $node)
+    public function previewAction(NodeAddress $node)
     {
         $nodeAddress = $node;
 
-        $inBackend = !$nodeAddress->isInLiveWorkspace();
-        $visibilityConstraints = $this->createVisibilityConstraints($inBackend);
```

Comment:
do we still need the method "createVisibilityConstraints"?

Reply:
No, that's why I removed it :)